riva/clients/asr/streaming_recognize_client.cc

```diff
@@ -139,6 +144,16 @@ StreamingRecognizeClient::GenerateRequests(std::shared_ptr<ClientCall> call)
       *(speech_context->mutable_phrases()) = {boosted_phrases_.begin(), boosted_phrases_.end()};
       speech_context->set_boost(boosted_phrases_score_);
 
+      // Set the endpoint parameters
+      // Get a mutable reference to the Endpointing config message
+      auto* endpointing_config = config->mutable_endpointing_config();
+      
+      endpointing_config->set_start_history(start_history_);
+      endpointing_config->set_start_threshold(start_threshold_);
+      endpointing_config->set_stop_history(stop_history_);
+      endpointing_config->set_stop_history_eou(stop_history_eou_);
+      endpointing_config->set_stop_threshold(stop_threshold_);
```

Comment:
so, when value is -1, it will land here. Let's add a check to not call these, if the value is -1, that should filter it out.